jaypeasee/whats-cookin
# Final Refactor

# Pull Request Template:


#### What's this PR do?
1. Cleans up styling consistencies specifically in the class files.
1. Renames some class method names for better semantics.
1. Moved `index.html` and `styles.css` to the root of the project director to fix Github pages issue.


#### Where should the reviewer start?
* Review `user.js`, `pantry.js`, and `recipe.js`
* Review `index.html` and how the script  and stye files are now linked.


#### How should this be manually tested?
1. Try to break the page! Click all the buttons.
1. Run `npm test` one more time to make sure all the tests are still passing.


#### Any background context you want to provide?
* This may be the last PR. 
* Watch out for potential merge conflicts with the html file.
